### DISCUSSION TITLE
Is there any detailed guide for configuration?

### DISCUSSION THREAD

**Discussion by flyegg** [qdm12/ddns-updater#430] [Q&A]
I have a problem when trying the first step "1. Create a directory of your choice, say data with a file named config.json inside."

I can find a window inside docker manager named terminal, but commands like mkdir or ls won't work. is there anything wrong or is there any detailed guide for newbie? Where can I type in those commands?

Best regards

**Comment by qdm12** [comment]
Try accessing your machine with a "terminal" or "ssh terminal". Or you can create a directory (same as `mkdir`) and change its ownership/permission. I've updated the instructions to be a bit clearer for the directory and config.json file creation in cf184070b8ecc954bd9a57b8a59448ce945de2ed ([rendered version](https://github.com/qdm12/ddns-updater/?tab=readme-ov-file#container))

If you still find difficulties, let me know and I'll try to help/clarify the readme
